### ISSUE TITLE
notie.confirm

### ISSUE THREAD

**Issue by globz** [jaredreich/notie#100]
Hi,

I would like to know if there's a way to avoid the following scenario, when I use `notie.confirm`, I give the user a choice between Yes and Delete, `Yes =>` will resume and load whatever is inside localStorage & `Delete =>` will delete whatever is inside localStorage.... but if you miss click outside of the confirm box it will always default to delete.  Is there any ways to change this behavior without going through a series of `notie.alert` just to make that the user did not click outside the confirm box by mistake? 

Basically a way to force the user to click YES OR DELETE but unable to dismiss the confirm box so this won't trigger the default behavior or going for the cancel function.

Thank you

**Comment by jaredreich** [comment]
Have you read the README? Scroll to Options > General and:

```js
backgroundClickDismiss: false
```

**Comment by globz** [comment]
Yes I did but somehow totally missed the specified option above. Thank you and sorry about the that.

**Comment by jaredreich** [comment]
Perhaps it wasn't documented well enough. Glad your problem is resolved. Best of luck.
